src/utils/http/fetchAPI.js

```diff
@@ -0,0 +1,56 @@
+import APIError from 'utils/http/errors/ApiError';
```

Comment:
better be consistent with naming -- since the component is called `ApiError`, better change `APIError` to `ApiError`

Reply:
I renamed it the other way around